.github/workflows/deploy.yml

```diff
@@ -0,0 +1,20 @@
+name: Deploy to WordPress.org
+on:
+  release:
+    types: [released]
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+jobs:
+  release:
+    name: New release to WordPress.org
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build # Remove or modify this step as needed
+        run: composer install --no-dev
+      - name: Push to WordPress.org
+        uses: 10up/action-wordpress-plugin-deploy@stable
+        env:
```

Comment:
You'll need to add the "slug" in this config, or rename the repo to use lowercase letters - the deploy action is case-sensitive.